src/spec.c

```diff
@@ -2365,7 +2365,7 @@ void Indexes_UpgradeLegacyIndexes() {
 
 void Indexes_ScanAndReindex() {
   if (!reindexPool) {
-    reindexPool = redisearch_thpool_create(1, DEFAULT_PRIVILEGED_THREADS_NUM, LogCallback);
+    reindexPool = redisearch_thpool_create(1, DEFAULT_PRIVILEGED_THREADS_NUM, LogCallback, "reindex");
```

Comment:
the thepool name is limited to 10 charchters. Also i think that the fact that it's a thpool suggests that it's done in the bg

Reply:
Isn't the limitation arbitrary? we can increase that if we like no?
"indexing" is also fine I guess

Reply:
No, it's limited by ` prctl(PR_SET_NAME)`

Reply:
https://man7.org/linux/man-pages/man2/prctl.2.html
> PR_SET_NAME (since Linux 2.6.9)
>               Set the name of the calling thread, using the value in the
>               location pointed to by (char *) arg2.  The name can be up
>               to 16 bytes long, including the terminating null byte.